controller/loadBasicPokemonDetails.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+let fs = require('fs');
+let basicPokemonDetail = require('../models/basicPokemonDetail.js');
+let pokemonListPath = require('../resources/json/pokemonlist.json'); 
+
+function loadBasicPokemonDetails() {
+    console.log('Loading Basic Pokemon Details to MongoDB');
+    let data = JSON.stringify(pokemonListPath, function(key,value) {
+        var pokemonID, pokemonName;
+        if(key != 'name' && typeof value.name != 'undefined') {
```

Comment:
It's the correct way of checking it, actually :) Just please use the triple operators instead :) makes more sense to me (at least for the `!==` operation), and use double quotations as in https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/typeof

If you wonder what the difference between `!==` and `!=` is, read http://www.c-point.com/javascript_tutorial/jsgrpComparison.htm and http://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons?page=1&tab=votes#tab-top